user_interaction.py

```diff
@@ -8,65 +8,58 @@ class MessageScope(Enum):
     PASSIVE = auto()
 
 
-prepared_messages = {
-    MessageScope.GLOBAL: [],
-    MessageScope.ACTIVE: [],
-    MessageScope.PASSIVE: []
-}
+class UserInteraction():
+    def __init__(self):
```

Comment:
should contain reference to Context in future

Reply:
Done

Reply:
It should be add in the next general commit